### ISSUE TITLE
Expand DID Configuration to handle more DID-related config items

### ISSUE THREAD

**Issue by csuwildcat** [decentralized-identity/.well-known#37]
As we look at how to make Credential Manifests discoverable, we have an opportunity to reuse the DID Configuration resource as the place to do that, as well as the things related to it. This might include a list of CMs (or references to them), details about issuance protocols (OIDC endpoints, DIDComms connection details, etc.). I think reusing this for more config stuff makes sense, given it is appropriately named/situated.

**Comment by OR13** [comment]
The current resource structure:

```json
{
  "@context": "https://identity.foundation/.well-known/contexts/did-configuration-v0.0.jsonld",
  "entries": [ VC_DATA_MODELs ]
}
```

Proposed structure:

```json
{
  "@context": "https://identity.foundation/.well-known/contexts/did-configuration-v0.0.jsonld",
  "linked_dids": [ VC_DATA_MODELs ],
  "supported_protocols": [ PROTOCOLs],
  "credential_manifests": [ CREDENTIAL_MANIFESTS]
}
```

**Comment by OR13** [comment]
potential issue is that the supported protocols implicitly apply to all linked dids...

**Comment by OR13** [comment]
Link to related hackmd...https://hackmd.io/QCJ3WDcLSXWp8t3hAQ387A

**Comment by csuwildcat** [comment]
> potential issue is that the supported protocols implicitly apply to all linked dids...

Good point - I wonder if there would need to be something in the issuance flow that specifies which DID the user wants to have a given cred issued from?

**Comment by OR13** [comment]
We can resolve this issue by adding a member of the `supported_protocols` objects called `linked_dids`... which tells you which linked did support which protocols... in the future... this kind of thing should be discoverable from the DID... not some well known centralized resource...

**Comment by OR13** [comment]
protocol data model...

```
name: string
documentation: URI
version: string
```

additional properties false, really don't want people to overload that object.

**Comment by csuwildcat** [comment]
Resolved:

- Other specs will define the schema-based objects this spec contains, and we need to define those objects before we start adding them to this spec
- Work on Credential Manifest spec
- Work on PE supported formats feature for inclusion as `supported_formats`

**Comment by OR13** [comment]
well known did configuration now blocked by:

https://github.com/decentralized-identity/presentation-exchange/issues/94

**Comment by OR13** [comment]
I would only be in favor of adding URLs at this time, this issue is going to stick aroudn until we decide which URLs to add or we close this issue :)
